metrics/storage/blogbench.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
We should consider adding a call to this script in https://github.com/kata-containers/tests/blob/master/.ci/run_metrics_PR_ci.sh as well I believe.

Reply:
@grahamwhaley, I just added it